release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1789,6 +1789,16 @@ Workaround: Use OpenShift OAuth for authentication. (link:https://bugzilla.redha
 *  The China (Nanjing) and UAE (Dubai) regions of Alibaba Cloud International Portal accounts do not support installer-provisioned infrastructure (IPI) installations. The China (Guangzhou) and China (Ulanqab) regions do not support a Server Load Balancer (SLB) if using Alibaba Cloud International Portal accounts and, therefore, also do not support IPI installations.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=2048062[*BZ#2048062*])
 
+* There is currently a known issue. The *Knative Serving - Revision CPU, Memory, and Network usage* and *Knative Serving - Revision Queue proxy Metrics* dashboards are visible to all the namespaces, including those that do not have Knative services. They should show only with namespaces that have Knative services. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2056682[BZ#2056682])
```

Comment:
```suggestion
* Currently, the *Knative Serving - Revision CPU, Memory, and Network usage* and *Knative Serving - Revision Queue proxy Metrics* dashboards are visible to all the namespaces, including those that do not have Knative services. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2056682[BZ#2056682])
```

**Observation/Question**:

-  No need to mention this line "There is currently a known issue." for every issue. You can also say "Currently,". Please refer to the **Example doc text** on the [RH supplementary style guide](https://redhat-documentation.github.io/supplementary-style-guide/#release-notes) for **Known issues**. (Global comment)
- I am not sure we need to include this expected behavior in the Known issue: "They should show only with namespaces that have Knative services." We can remove this. Please refer to the **Example doc text** on the [RH supplementary style guide](https://redhat-documentation.github.io/supplementary-style-guide/#release-notes) for **Known issues**. Suggest to state just the problem and workaround if any. Consult with the SMEs once about this and do the needful as per the RH writing guidelines.
- If there is no workaround, please add this sentence at the end of the Known issue text: "There is currently no workaround for this issue." (Global comment)
- If there is a workaround, please mention it like this:
"Workaround: _Workaround Text_" (Global comment)

Reply:
Lol. @Srivaralakshmi @rishumehra 
- I just finished adding the "Currently there is..." phrase because the SSG template for known issues includes it. 
- The SSG template for known issues doesn't include the "There is no known..." phrase, although the example it provides does. It probably helps (or doesn't hurt) to mention this so that customers looking for a workaround can quickly understand if there isn't one.

Reply:
There is no workaround for these given Known Issues. Although, for known issues, if a workaround is not written then it is understood that there is on workaround with that issue. So I'll drop adding the "There is currently no workaround for this issue."  and incorporate the "Currently" part.